packages/suite/src/components/onboarding/GlobalWebNavigation/index.messages.tsx

```diff
@@ -0,0 +1,21 @@
+import { defineMessages } from 'react-intl';
```

Comment:
global navigation is already in suite core

Reply:
Yes, should be removed.